avocado_vt/test.py

```diff
@@ -223,7 +224,9 @@ def __safe_env_save(self, env):
         try:
             env.save()
         except Exception as details:
-            if hasattr(stacktrace, "str_unpickable_object"):
+            try:
+                pickle.dumps(env.data)
+            except Exception:
                 self.log.warn("Unable to save environment: %s",
                               stacktrace.str_unpickable_object(env.data))
             else:    # TODO: Remove when 36.0 LTS is not supported
```

Comment:
The "TODO" comments need to be removed as it will be used in the future. And what type of exception it catch during env.save?

Reply:
I assume 36.0 LTS is still used on RHEL6.9. The main logic here is that env.data may be one pickable object, maybe not in some cases,so need carefully manage this.